templates/details/_side-nav.html

```diff
@@ -74,62 +74,73 @@ <h2 class="p-modal__title" id="modal-title">{{ package.name }}</h2>
       <li class="p-list__item">
-        <a href="{{ package["store_front"]["metadata"]["issues"][0] }}"><i class="p-icon--bug"></i>&nbsp;&nbsp;Submit a bug</a>
+        <a href="{{ package["store_front"]["metadata"]["source"] }}"><i class="p-icon--github-grey"></i>&nbsp;&nbsp;Charm repository</a>
       </li>
-    {% elif package.result["bugs-url"] %}
+    {% endif %}
+    {% if package["result"]["links"]["website"] %}
+      <ul class="p-list u-no-margin--bottom">
+        {% for website in package["result"]["links"]["website"] %}
+        <li class="p-list__item">
+          <a href="{{ website }}"><i class="p-icon--exposed"></i>&nbsp;&nbsp;Charm product page</a>
+        </li>
+        {% endfor %}
+      </ul>
+    {% elif package.result.website %}
       <li class="p-list__item">
-        <a href="{{ package.result["bugs-url"] }}"><i class="p-icon--bug"></i>&nbsp;&nbsp;Submit a bug</a>
+        <a href="{{ package.result.website }}"><i class="p-icon--exposed"></i>&nbsp;&nbsp;Charm product page</a>
       </li>
     {% endif %}
-    {% if package.result.contact %}
+    {% if juju_version %}
       <li class="p-list__item">
-        <a href="{{ package.result.contact }}"><i class="p-icon--contact"></i>&nbsp;&nbsp;Contact</a>
+        <h5 class="side-nav-muted-text u-text--muted">JUJU VERSION</h5>
+        <p class="u-no-margin--bottom">{{ juju_version }}</p>
       </li>
     {% endif %}
   </ul>
   <hr class="p-separator--shallow" />
 {% endif %}
-<!-- Once `links` is fully ready we can remove the `request.args` check -->
-{% if package["result"]["links"] and request.args.get("show_metadata_links") %}
-  {% if package["result"]["links"]["website"] %}
-  <h2 class="p-heading--5 u-no-margin--bottom">Websites</h2>
-    <ul class="p-list">
-      {% for contact in package["result"]["links"]["website"] %}
-      <li><a href="{{ website }}">{{ website }}</a></li>
-      {% endfor %}
-    </ul>
-    <hr class="p-separator--shallow" />
-  {% endif %}
-  {% if package["result"]["links"]["contact"] %}
-    <h2 class="p-heading--5 u-no-margin--bottom">Contact</h2>
+{% if package["result"]["links"]["contact"] or package["store_front"]["metadata"]["maintainers"] or package["store_front"]["metadata"]["issues"] or package.result["bugs-url"] %}
+  <h4 class="p-heading--5">Contacts</h4>
     <ul class="p-list">
-      {% for contact in package["result"]["links"]["contact"] %}
-      <li><a href="{{ contact }}">{{ contact|replace('mailto:', '') }}</a></li>
-      {% endfor %}
+      {% if package["result"]["links"]["contact"] %}
+        <h5 class="side-nav-muted-text u-text--muted">MAINTAINERS</h5>
+        <ul class="p-list contact-lists">
+          {% for contact in contacts %}
+          <li class="p-list__item"><a href="mailto:{{ contact.email }}"><i class="p-icon--user-grey"></i>&nbsp;&nbsp;{{ contact.name }}</a></li>
+          {% endfor %}
+        </ul>
+      {% elif package["store_front"]["metadata"]["maintainers"] %}
```

Comment:
I'm not sure this should be an else if - @lukasSerelis wdyt? Should we be showing both official contact details AND maintainers? Or one or the other? I would have thought both as the main contact may be "Canonical Data Team" but the maintainers could be multiple individuals?

Reply:
The end user just cares to find out where to get help/give suggestions/contribute, so as long as there are relevant contact details it shouldn't matter too much what we display.

That being said, adding more contact details than necessary might not make it clear who to contact first, making it more difficult than we think. I'd say the elif here is justified in that case.